.gitignore

```diff
@@ -13,3 +13,6 @@ node_modules
 test/benchmarks/.tmp
 coverage.lcov
 test-suite-output.tap
+.pytest_cache/
+**/__pycache__/
+*.pyc
```

Comment:
We have previously put files and folders in separate sections in this file. If `*.pyc` is a file, it should go up into the file-section above.

I don't have any hard feelings whether or not we should continue this practice, but until/if we change it, we should probably continue it 🙂

Reply:
No problem at all. :) I'll change it. Thanks for letting me know.